src/AutoBuild.jl

```diff
@@ -19,7 +19,7 @@ function autobuild(dir::AbstractString, src_name::AbstractString,
         run_travis_busytask = true
         travis_busytask = @async begin
             # Don't let Travis think we're asleep...
-            info("Brewing a pot of coffee for Travis...")
+            Compat.@info("Brewing a pot of coffee for Travis...")
```

Comment:
Why use `Compat.@info` here but change the other `info` uses to `println`s?

Reply:
The other uses were doing `info(io, ...)`, of which there isn't a direct compat for the new logging system (i.e. logging to a specific io stream). I tried to change as little as possible to allow package authors to eventually redesign overall logging as they see fit.

Reply:
We could make those `printstyled("INFO: ...", color=Base.info_color())` but that's probably overkill.

Reply:
Better to just get 0.7 out already so we can all do proper logging 😉